Documentation/deployment view

TomasArguelles opened the pull request:
Mismo que en los demás, si veis algo que cambiar avisadme y lo cambio

TomasArguelles committed:
Add files via upload

Diagrama de funcionamiento basico

TomasArguelles committed:
Update 07_deployment_view.adoc

Esta cosa sonaba más complicada de lo que era _-_

TomasArguelles committed:
Update 07_deployment_view.adoc

typos

acg96 commented:
Aquí la aplicación dechat era bueno indicar que se alojará en un servidor una vez esté en desarrollo y en local mientras está en test y explicaría un poco los requisitos que se necesitan para que corra la aplicación (necesita Node.js y angular/cli y el sistema operativo no influiría porque es multiplataforma), pero se necesita una infrastructura para que esté arrancada. Respecto a los servidores de SOLID explicar un poco por encima en qué consisten, es decir, que son servidores que cualquiera puede configurar en base a unas especificaciones y que la gente decidirá donde almacenar sus POD.
Luego hay un fallo, cuando se envía un mensaje, este se guarda en el propio POD pero no se guarda en el del otro como he visto que se indica con una flecha. Lo que se hace es notificar a la otra persona.

Y debería seguirse el esquema de niveles de infraestructura que se indica en https://docs.arc42.org/section-7/ (son 2, el primero para mostrar así a grandes rasgos como indicaste y luego el 2o que trata de explicar más por dentro cada parte importante, en este caso sobre todo como desplegar la app) que obviamente no se sabe donde están los servidores SOLID pero se puede especificar lo que dije antes que hay varios, etc...

TomasArguelles committed:
Add files via upload

añadido diagrama arreglado

TomasArguelles committed:
Update 07_deployment_view.adoc

update con diagrama corregido

TomasArguelles committed:
Delete Diagram 2019-03-30 19-51-56.png

borrado diagrama antiguo

TomasArguelles committed:
Update 07_deployment_view.adoc

Me atasqué absolutamente con el segundo nivel de la infraestructura, no sé qué poner
